core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -26,9 +26,6 @@
 import lombok.Setter;
 import lombok.SneakyThrows;
 import org.apache.commons.codec.digest.DigestUtils;
-import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
```

Comment:
Note that there seem to be a lot of import movements. I'm not sure why this is suddenly happening, but I'm confident that the `.editorconfig` rules we have set up are being respected:

```
[*.java]
indent_style = space
indent_size = 4
# Never use star imports
ij_java_names_count_to_use_import_on_demand = 99
ij_java_class_count_to_use_import_on_demand = 99
ij_java_layout_static_imports_separately = true
```

Maybe recent edits to these files have not been picking up the rules?